### PR TITLE
github: move pull request template tip into comment

### PR DESCRIPTION
**Pull request by Rustin-Liu**
Signed-off-by: Rustin-Liu <rustin.liu@gmail.com>

###  What have you changed?

Modify pull request template.

###  What is the type of the changes?

- Misc (other changes)

###  How is the PR tested?
None

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

None

###  Does this PR affect `tidb-ansible`?

None

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -31,20 +31,21 @@ Pick one of the following and delete the others:
 <!--
 Please select the tests that you ran to verify your changes:
```
Comment (breeswish) [inline .github/pull_request_template.md]: Change it to "Pick one of the following and delete the others:" as well?

Comment (Rustin-Liu) [reply .github/pull_request_template.md]: I think this is multiple choice not a single choice.

**Pull request merged**